### ISSUE TITLE
This backend doesn't support absolute paths.

### ISSUE THREAD

**Issue by BojanKogoj** [Hipo/drf-extra-fields#52]
When using `Base64ImageField` it throws above exception, I only use it in responses. It works fine locally, but not on the server which is Heroku + S3. I think the problem is with [file.path](https://github.com/Hipo/drf-extra-fields/blob/master/drf_extra_fields/fields.py#L74) which exists locally only.

I'm using Django 2.0.2 and DRF 3.7.7

**Comment by yigitguler** [comment]
It seems like a django-storages bug.

**Comment by BojanKogoj** [comment]
Not exactly. They don't implement `.path` for non local files. I managed to get it working with a small change in this library
```python
try:
    return base64.b64encode(f.read()).decode()
except Exception:
    raise IOError("Error encoding file")
```

Any downside to this?

**Comment by yigitguler** [comment]
Oh, it is in the read mode. Yeah, an improvement might be needed. Expecting a PR then :)

**Comment by yigitguler** [comment]
Fixed on 1.2.0
